js/client-battle.js

```diff
@@ -396,6 +396,11 @@
 						if (move.id === 'Struggle' || move.id === 'Recharge') pp = '&ndash;';
 						if (move.id === 'Recharge') move.type = '&ndash;';
 						if (name.substr(0, 12) === 'Hidden Power') name = 'Hidden Power';
+						// Update Judgment's type to match that of the Plate type, if any.
+						var item = Tools.getItem(switchables[pos].item);
```

Comment:
You can check for it after checking `move.name`. I guess either way is fine, but that's how I'd do it.